README.md

```diff
@@ -1,43 +1,66 @@
 # Lua on Unikraft
```

Comment:
Provide a short description on what is the purpose of the application, similar to [here](https://github.com/unikraft/app-nginx)